opencog/reasoning/RuleEngine/rules/pln/evaluation-to-member.scm

```diff
@@ -0,0 +1,64 @@
+; ============================================================================= 
+; GeneralEvaluationToMemberRule
+;	EvaluationLink (pred D (ListLink B C))
+;		becomes
+;	MemberLink( B (SatisfyingSetLink( Variable $X 
+;		(EvaluationLink (pred D (ListLink X C))))))
+; -----------------------------------------------------------------------------
+
+(define pln-rule-evaluation-to-member
+	(BindLink
+		(ListLink
+			(VariableNode "$B")
+			(VariableNode "$C")
+			(VariableNode "$D")
+			(TypedVariableLink
+				(VariableNode "$D")
+				(VariableTypeNode "PredicateNode")))
+		(ImplicationLink
+			(EvaluationLink
+				(VariableNode "$D")
```

Comment:
The predicate is taking only two arguments, you should define a function similar to https://github.com/opencog/opencog/blob/master/opencog/python/pln/rules/direct_evaluation_rules.py#L305 , because the predicate could have 1, 2, or 3 arguments in most cases, but there is nothing preventing it from having more than that